src/components/Settings/Settings.js

```diff
@@ -105,6 +104,29 @@ class Settings extends React.Component {
             });
         }
     }
+    changeBack = () => {
+        let backgroundImage = document.getElementById('backgroundImgUrl').value;
+        console.log(backgroundImage);
+            if(cookies.get('loggedIn')===null||cookies.get('undefined')){
+                if(backgroundImage){
+                localStorage.setItem('anonymousImage',backgroundImage);
+                }
+                else{
+                    localStorage.setItem('anonymousImage','');
+                }
+            }
+            else {
+                /* eslint-disable */
```

Comment:
What was the error that you disabled eslint? :thinking:

Reply:
It was not considering the else statement and was giving the error of lonely if statement.